### PR TITLE
Change prior notice field type (GTFS-lib update)

### PR DESCRIPTION
**Pull request by br648**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

 GTFS-lib update to include the field type change to prior notice fields in booking rules.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -269,7 +269,7 @@
         <dependency>
             <groupId>com.github.conveyal</groupId>
             <artifactId>gtfs-lib</artifactId>
-            <version>9ad47be</version>
+            <version>084cb3f</version>
```
Comment (philip-cline) [inline pom.xml]: Should this be this now?
```
            <version>34a75aa</version>
```

**Review by philip-cline — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (br648) [reply pom.xml]: Yes and no. I only made unit test changes under this version, so the functionality is still the same.

Comment (br648) [reply pom.xml]: I've updated.

**Review by miles-grant-ibigroup — APPROVED** [review]
Working thanks

**Pull request merged**